db/build.rs

```diff
@@ -0,0 +1,5 @@
+// generated by `sqlx migrate build-script`
+fn main() {
+    // trigger recompilation when a new migration is added
+    println!("cargo:rerun-if-changed=migrations");
+}
```

Comment:
Ahh! This should fix the issue where we would change a mirgation and cargo wouldn't re-compile and realize the change lol

Now it will!